content/en/docs/reference/using-api/client-libraries.md

```diff
@@ -68,12 +68,12 @@ their authors, not the Kubernetes team.
 | Ruby                 | [github.com/kontena/k8s-client](https://github.com/kontena/k8s-client) |
 | Rust                 | [github.com/clux/kube-rs](https://github.com/clux/kube-rs) |
 | Rust                 | [github.com/ynqa/kubernetes-rust](https://github.com/ynqa/kubernetes-rust) |
+| Scala                | [github.com/joan38/kubernetes-client](https://github.com/joan38/kubernetes-client) |
 | Scala                | [github.com/doriordan/skuber](https://github.com/doriordan/skuber) |
```

Comment:
nit:
```suggestion
| Scala                | [github.com/doriordan/skuber](https://github.com/doriordan/skuber) |
| Scala                | [github.com/joan38/kubernetes-client](https://github.com/joan38/kubernetes-client) |
```
(d sorts before j)

The overall list is not as sorted as it could be, I admit!